src/codeu/chat/server/Model.java

```diff
@@ -182,7 +182,7 @@ public void updateMessageLink(Message message) {
 	  // EDIT - Malik Graham
 	  // Updating the next message for the last message
       try {
-    	  ds.updateLastMessage(message);
+    	  ds.updateNextMessage(message);
```

Comment:
Why would these functions throw an exception? Could we handle them a little better than just printing the stack trace?